portfolio/src/main/webapp/js/comments.js

```diff
@@ -108,3 +123,32 @@ function postComment(event) {
           "alert-danger");
     });
 }
+
+/**
+ * Gets a function that sends a comment delete request.
+ * @param {number} id
+ * @return Returns a function that deletes the comment associated with an ID.
+ */
+function getCommentDeleter(id) {
```

Comment:
Use "create" rather than "get". "get" implies that something already exists and you are merely retrieving it.
```suggestion
 * Creates a function that sends a comment delete request.
 * @param {number} id
 * @return Returns a function that deletes the comment associated with an ID.
 */
function createCommentDeleter(id) {
```

Reply:
Agreed. Changing.